lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -2746,6 +2748,13 @@ bool TFGraphLowering::lowerTFGraphOrFunction(
   // we sort the device IDs first.
   SmallVector<DeviceId, 8> deviceIds(deviceInfo.getUsedDeviceIds().begin(),
                                      deviceInfo.getUsedDeviceIds().end());
+  // TODO: unify these two code paths
+  if (TFUseDeviceStack) {
+    assert(deviceIds.size() == 1);
+    llvm::errs() << "lowering: " << deviceIds[0] << "\n";
```

Comment:
If this is for debugging, it should be dumped to tf-dump stream or use `LLVM_DEBUG(llvm::dbgs() << "...")`.

Reply:
Ah I meant to delete this line. Thanks for the catch!